src/bindings/python/test.py

```diff
@@ -38,7 +38,7 @@
 import unittest
 
 test_basedir = 'test'
```

Comment:
I built this branch with no problem on my macOS, the tests compiled and ran fine too. However, in my built this test directory was not generated, which makes me think that I may have missed something?

Reply:
I think this is possibly referring to `liblx/src/bindings/python/test`